src/hydra_zen/structured_configs/_implementations.py

```diff
@@ -607,6 +611,24 @@ def builds(
                 _utils.field(default=just(target), init=False),
             ),
         ]
+    elif hydra_meta:
```

Comment:
Is it necessary to handle the case where both `hydra_partial` is `True` and `hydra_meta` is not `None`?

Reply:
Yep! I definitely do need to handle both of these cases. This is just a crude WIP right now. 

I plan to refactor how we handle `hydra_partial=True` so that it also looks for `_true_target_` instead of `_partial_target_`. I want to converge on a common interface for any/all post-processing functions that `hydra-zen` ships.